nexus/reconfigurator/execution/src/lib.rs

```diff
@@ -611,50 +611,36 @@ fn register_reassign_sagas_step<'a>(
                         .into();
                 };
 
-                // Re-assign sagas, but only if we're allowed to.  If Nexus is
-                // quiescing, we don't want to assign any new sagas to
-                // ourselves.
-                let result = saga_quiesce.reassign_if_possible(async || {
-                    // For any expunged Nexus zones, re-assign in-progress sagas
-                    // to some other Nexus.  If this fails for some reason, it
-                    // doesn't affect anything else.
-                    let sec_id = nexus_db_model::SecId::from(nexus_id);
-                    let reassigned = sagas::reassign_sagas_from_expunged(
-                        opctx, datastore, blueprint, sec_id,
-                    )
-                    .await
-                    .context("failed to re-assign sagas");
-                    match reassigned {
-                        Ok(needs_saga_recovery) => (
-                            StepSuccess::new(needs_saga_recovery).build(),
-                            needs_saga_recovery,
-                        ),
-                        Err(error) => {
-                            // It's possible that we failed after having
-                            // re-assigned sagas in the database.
-                            let maybe_reassigned = true;
-                            (
-                                StepWarning::new(false, error.to_string())
-                                    .build(),
-                                maybe_reassigned,
-                            )
+                // Re-assign sagas.
+                Ok(saga_quiesce
+                    .reassign_sagas(async || {
+                        // For any expunged Nexus zones, re-assign in-progress
+                        // sagas to some other Nexus.  If this fails for some
```

Comment:
I realize this comment was here before, but can we refine "some other Nexus" to "ourself"?